packages/lib/gpcircuits/src/proto-pod-gpc.ts

```diff
@@ -376,9 +388,29 @@ export class ProtoPODGPC {
     );
   }
 
-  private static circuitNameForParams(
-    params: ProtoPODGPCCircuitParams
-  ): string {
+  /**
+   * Calculates the merged set of parameters which meets the unified (maximum)
+   * requirements of both inputs.
+   *
+   * @param rp1 first set of required parameters
+   * @param rp2 second set of required paremeters
+   * @returns unified (maximum) parameters
+   */
+  public static mergeRequiredParams(
```

Comment:
Where is this used?

Reply:
It's not in this PR yet.  It's used in lib/gpc which is what I was working on when I discovered this issue.